.claude/commands/setup/setup-linting.md

```
@@ -0,0 +1,76 @@
+6. **Package.json Scripts**
+   ```json
+   {
+     "scripts": {
+       "lint": "eslint src --ext .ts,.tsx",
+       "lint:fix": "eslint src --ext .ts,.tsx --fix",
+       "format": "prettier --write src"
+     }
+   }
+   ```
+
+Remember to customize rules based on team preferences and gradually enforce stricter standards.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)